pkg/proxy/ipvs/proxier.go

```diff
@@ -523,7 +523,7 @@ func NewDualStackProxier(
 		exec, syncPeriod, minSyncPeriod, filterCIDRs(true, excludeCIDRs), strictARP,
 		tcpTimeout, tcpFinTimeout, udpTimeout, masqueradeAll, masqueradeBit,
 		localDetectors[1], hostname, nodeIP[1],
-		nil, nil, scheduler, nodePortAddresses, kernelHandler)
+		recorder, healthzServer, scheduler, nodePortAddresses, kernelHandler)
```

Comment:
The `recorder` definitely needs to be passed in here. I'm not sure about `healthzServer` yet. The proxier does do a nil checks for it so it shouldn't result in a panic when nil. 

If we pass the same healthz server to both IPv4  and IPv6 proxiers, that means healtz server will report "healthy" even if only one of the proxiers are healthy but the other one is down. I'm not sure that's the behavior we want.

Reply:
I assumed that this was correct, as that is how it is currently implemented in the iptables proxier: https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/iptables/proxier.go#L360-L369